src/backends/caffe/caffeinputconns.h

```diff
@@ -385,10 +385,14 @@ namespace dd
 		return;
 	      }
 	      // create db
-	      if (!this->_unchanged_data)
-		images_to_db(_uris,_model_repo + "/" + _dbname,_model_repo + "/" + _test_dbname);
-	      else images_to_db(_uris,_model_repo + "/" + _dbname,_model_repo + "/" + _test_dbname,
-				"lmdb",false,"");
+				// Check if the indicated uri is a folder
+				bool dir_images = true;
+				bool exists= fileops::file_exists(_uris.at(0), dir_images);
```

Comment:
Not sure what this variable is used for, as I don't see it elsewhere in the diff. Do we need it ?

Reply:
Indeed, it's not used, I deleted it